dex/src/main/scala/com/wavesplatform/dex/actors/address/BatchOrderCancelActor.scala

```diff
@@ -9,40 +9,48 @@ import com.wavesplatform.dex.domain.order.Order
 import com.wavesplatform.dex.domain.utils.ScorexLogging
 import com.wavesplatform.dex.error
 
+import scala.collection.immutable.ListMap
 import scala.concurrent.duration.FiniteDuration
 
 class BatchOrderCancelActor private (
-  orderIds: Set[Order.Id],
+  orderIds: List[Order.Id],
   source: Command.Source,
   processorActor: ActorRef,
   clientActor: ActorRef,
-  timeout: FiniteDuration,
-  initResponse: Map[Order.Id, OrderCancelResult]
+  timeout: FiniteDuration
 ) extends Actor
     with ScorexLogging {
 
   import BatchOrderCancelActor._
   import context.dispatcher
 
-  orderIds.foreach(processorActor ! CancelOrder(_, source))
+  final private val uniqueOrderIds: Set[Order.Id] = orderIds.toSet
```

Comment:
`final` is unnecessary here, because it is `private`

Reply:
ok